### PR TITLE
AMA-428 - add checkboxes and new status -> uploading

### PR DESCRIPTION
**Pull request by NgoniChoga**

### CODE REVIEW

File: admin-home.php [hunk 1]
```diff
@@ -82,7 +82,8 @@
     'brightspace_url' => $tool['brightspace_url'],
     'brightspace_log_url' => $tool['brightspace_log_url'],
     'vula_url' => $tool['vula_url'],
-    'jira_url' => $tool['jira_url']
+    'jira_url' => $tool['jira_url'],
+    'conversion_test' => $LAUNCH->ltiRawParameter('conversion_test', false)
```
Comment (npmali) [inline admin-home.php]: In order for the 'Test conversion' checkbox to be checked on default, add "custom_" in front of the 'conversion_test' ltiRawParameter. 

So this line should be:

`'conversion_test' => $LAUNCH->ltiRawParameter('custom_conversion_test', false)`

File: templates/admin-body.html [hunk 2]
```diff
@@ -145,7 +146,20 @@ <h2>Admin <small>(Migrate list of sites)</small></h2>
...
+                                            <div class="col-md-12 col-xs-12">
+                                                <div class="form-check">
+                                                    <label class="form-check-label" for="conversion_test">
+                                                    <input type="checkbox" class="form-check-input" id="conversion_test" {% if ($conversion_test ==true) { %}checked{% } %}>
+                                                    Test conversion</label>
+                                                </div>
+                                                <div class="form-check">
+                                                    <label class="form-check-label" for="enrol">
+                                                    <input type="checkbox" class="form-check-input" id="enrol">
+                                                    Enrol site owners and support staff</label>
```
Comment (npmali) [inline templates/admin-body.html]: I think you're missing text on this label. According to the JIRA the text should be: 

`Enrol site owners and support staff in the converted site`

**Review by npmali — CHANGES_REQUESTED** [review]

File: templates/admin-footer.html [hunk 3]
```diff
@@ -314,12 +316,9 @@
 
         $('#btnAccept').click(function(event) {
             event.preventDefault();
-            doPost('btnAccept', 'Adding sites ...');
-        });
-
-        $('#test_conversion').click(function(event) {
-            event.preventDefault();
-            doPost('test_conversion', 'Adding test sites ...');
+            var conversion_test = $('#conversion_test').is(':checked') ? 'test_conversion' : 'btnAccept';
```
Comment (TurRil) [inline templates/admin-footer.html]: Maybe change it to:
```
var btn_type =$('#conversion_test').is(':checked') ? 'test_conversion' : 'btnAccept';
```
So it is the same as line 344.

File: templates/admin-footer.html [hunk 4]
```diff
@@ -37,42 +37,44 @@
             $('#info').hide();
         }
 
-        function showError(a) {
-            $('#' + a).html('<i class="fa fa-exclamation"></i> Error').addClass('disabled').attr('disabled', true);
+        function showError(conversion_test) {
```
Comment (TurRil) [inline templates/admin-footer.html]: Change `conversion_test` to `btn` to make it more descriptive (I know `a` was not really descriptive enough for this)

File: templates/admin-footer.html [hunk 5]
```diff
@@ -37,42 +37,44 @@
...
+            $('#' + conversion_test).html('<i class="fa fa-exclamation"></i> Error').addClass('disabled').attr('disabled', true);
             $('#message').html('<p class="bg-danger">An error occurred while performing this action, please contact Admin.</p>');
         }
 
-        function submit_sites(sites, a, text) {
+        function submit_sites(sites, conversion_test, text, enrol) {
```
Comment (TurRil) [inline templates/admin-footer.html]: Change `conversion_test` to `btn`

File: templates/admin-footer.html [hunk 6]
```diff
@@ -37,42 +37,44 @@
...
             const re = /[{(]?[0-9A-F]{8}[-]?([0-9A-F]{4}[-]?){3}[0-9A-F]{12}[)}]?/i;
             
             if (sites.length > 0) {
-                $('#' + a).html('<i class="fa fa-cog fa-spin"></i>' + text).addClass('disabled').attr('disabled', true);
+                $('#' + conversion_test).html('<i class="fa fa-cog fa-spin"></i>' + text).addClass('disabled').attr('disabled', true);
```
Comment (TurRil) [inline templates/admin-footer.html]: Change `conversion_test` to `btn`

File: templates/admin-footer.html [hunk 7]
```diff
@@ -37,42 +37,44 @@
...
                                                         'sites': sites.map(x => x.match(re)[0]),
                                                         'term': $('#year').val(),
-                                                        'is_test': a == 'test_conversion' ? 1 : 0 }, function(result) {
+                                                        'is_test': conversion_test === 'test_conversion' ? 1 : 0,
+                                                        'enrol': enrol? 1: 0
+                }, function(result) {
                     hideHelp();
                     if (result.success === 1) {
-                        $('#' + a).html('<i class="fa fa-check"></i> Refreshing page ...');
+                        $('#' + conversion_test).html('<i class="fa fa-check"></i> Refreshing page ...');
```
Comment (TurRil) [inline templates/admin-footer.html]: Change `conversion_test` to `btn`

File: templates/admin-footer.html [hunk 8]
```diff
@@ -37,42 +37,44 @@
...
                         setTimeout(function() { location.reload(); }, 300);
                     } else {
-                        showError(a);
+                        showError(conversion_test);
```
Comment (TurRil) [inline templates/admin-footer.html]: Change `conversion_test` to `btn`

File: templates/admin-footer.html [hunk 9]
```diff
@@ -37,42 +37,44 @@
...
                 })
                 .always(function() {
                     hideHelp();
                 });
             }
         }
 
-        function doPost(a, text) {
-            submit_sites( $('#add_sites').val().split(/\r?\n/g).filter( (x) => { return x.length  > 2 }), a, text);
+        function doPost(conversion_test, text, enrol) {
```
Comment (TurRil) [inline templates/admin-footer.html]: Change `conversion_test` to `btn`

**Review by TurRil — CHANGES_REQUESTED** [review]

**Review by TurRil — APPROVED** [review]

**Review by TurRil — APPROVED** [review]

**Pull request merged**